src/app/components/events/events.component.html

```diff
@@ -35,13 +35,13 @@
                 </span>
               </p>
             </mat-card-content>
-            <mat-card-actions>
-              <a href="#" target="_blank">
-                <button color="primary" class="btn" disabled>
-                  <span>REGISTER</span>
-                </button>
-              </a>
-            </mat-card-actions>
+            <!--            <mat-card-actions>-->
+            <!--              <a href="#" target="_blank">-->
+            <!--                <button color="primary" class="btn" disabled>-->
+            <!--                  <span>REGISTER</span>-->
+            <!--                </button>-->
+            <!--              </a>-->
+            <!--            </mat-card-actions>-->
```

Comment:
Uhhhh!, I don't like to see commented code.

Reply:
Good